pkg/controller/pod/pod_controller.go

```diff
@@ -76,8 +77,23 @@ type ReconcilePolicy struct {
 // Reconcile reads that state of the cluster for a Pod object and makes changes based on the state
 func (r *ReconcilePolicy) Reconcile(request reconcile.Request) (reconcile.Result, error) {
 	log.V(1).Info("got a pod event in the controller")
```

Comment:
Add the pod name and namespace to a logger so whenever we log something it appears there.
`logger := log.WithName("Reconcile").WithValues("podName", request.Name, "podNamespace", request.Namespace)`

Then we do normal

`logger.V(1).Info("got a pod event in the controller")` 

and the function, an pod name and namespace will appear at every log.

Reply:
neat!